test/core/rules/std_test.py

```diff
@@ -60,8 +60,19 @@ def assert_rule_pass_in_sql(code, sql, configs=None):
         ("L005", "fail", "SELECT 1 ,4", "SELECT 1,4", None),
         ("L008", "pass", "SELECT 1, 4", None, None),
         ("L008", "fail", "SELECT 1,   4", "SELECT 1, 4", None),
+        ("L008", "fail", "SELECT 1,4", "SELECT 1, 4", None),
         ("L013", "pass", "SELECT *, foo from blah", None, None),
-        ("L013", "fail", "SELECT upper(foo), bar from blah", None, None),
```

Comment:
What's the thinking on removing this one? I wonder whether it's worth keeping to make sure function parsing is still picked up?

Reply:
🤦 Must've fat-fingered this one, nice catch!